### ISSUE TITLE
[Quest][Wetlands] A Grim Task

### ISSUE THREAD

**Issue by kudryashov-n** [Atlantiss/NetherwingBugtracker#563]
[//]: # (Enclose links to things related to the bug using http://wowhead.com or any other TBC database.)
[//]: # (You can use screenshot ingame to visual the issue.)
[//]: # (Write your tickets according to the format:)
[//]: # ([Quest][Azuremyst Isle] Red Snapper - Very Tasty!)
[//]: # ([NPC] Magistrix Erona)
[//]: # ([Spell][Mage] Fireball)
[//]: # ([Npc][Drop] Ghostclaw Lynx)
[//]: # ([Web] Armory doesnt work)

**Description**: 
Balagaras the Foul doesn't spawn at the place he suppose to be.
**Current behaviour**:
To sum up, i checked him three time for an hour and he wasn't still there.
**Expected behaviour**: 
He suppose to respawn with max 15 mins. 
**Server Revision**:

**Comment by aresbrutus** [comment]
This mob has a total of 3 spawn positions. This means that if he is not at spot 1, he might be at 2 or 3. Please read the quest comments and go look for him.
